homework/index.js

```diff
@@ -29,19 +29,145 @@
     });
     return elem;
   }
+  /* cSpell:disable */
+  // Creating the repository-details part
+  function createHyfRepoDetails(leftDiv, hyfRepo) {
+    const table = createAndAppend('table', leftDiv, { class: 'table' });
+    const tBody = createAndAppend('tbody', table);
+    const details = ['Repository: ', 'Description: ', 'Forks: ', 'Updated: '];
+    details.forEach(detail => {
+      const tr = createAndAppend('tr', tBody);
+      createAndAppend('td', tr, {
+        text: detail,
+        class: 'label',
+      });
+      createAndAppend('td', tr);
+    });
+    const secondTd = document.getElementsByTagName('td')[1];
+    createAndAppend('a', secondTd, {
+      href: hyfRepo.html_url,
+      target: '_blank',
+      text: hyfRepo.name,
+      class: 'repo-name',
+    });
+    if (hyfRepo.description === null) {
+      document.getElementsByTagName('tr')[1].setAttribute('class', 'hide-description');
+    } else {
+      document.getElementsByTagName('td')[3].innerText = hyfRepo.description;
```

Comment:
This works (`document.getElementsByTagName('td')[3]`) but what if we add a `td` in front of all of them, then the third one would not be description anymore, better to use a classname in this case since you always want the description on the `td` that is for the description.

Reply:
Yes, you are right... Thanks, I'll try to fix it.